scripts/convert-auroraV1-to-V2-postgres.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+set -xe
+# Script to convert a serverless v1 postgresql to serverless v2
+# Steps from https://aws.amazon.com/blogs/database/upgrade-from-amazon-aurora-serverless-v1-to-v2-with-minimal-downtime/
+
+DATABASE_IDENTIFIER=
+GREEN_DATABASE_IDENTIFIER=featureflags-dev-green
+SMALL_DATABASE_IDENTITER=featureflags-dev
+ACCOUNT_ID=
+REGION=us-east-1
+
+# Set AWS_PAGER to an empty string to disable the pager
+export AWS_PAGER=""
+
+echo 'creating new replication param groups'
+aws rds create-db-cluster-parameter-group \
+ --db-cluster-parameter-group-name aurora-postgres-with-replication \
+ --description 'Aurora Postgres 13 With Replication Enabled' \
+ --db-parameter-group-family aurora-postgresql13 \
+ --no-paginate
+
+aws rds modify-db-cluster-parameter-group \
+ --db-cluster-parameter-group-name aurora-postgres-with-replication \
+ --parameters 'ParameterName=rds.logical_replication,ParameterValue=1,ApplyMethod=pending-reboot' \
+ --no-paginate
+
+echo 'attaching new param group'
+
+aws rds modify-db-cluster \
+ --db-cluster-identifier $DATABASE_IDENTIFIER \
+ --db-cluster-parameter-group-name aurora-postgres-with-replication \
+ --apply-immediately \
+ --no-paginate
+
+echo "According to Amazon there needs to be a more then 10 minute delay between these too operations, so we wait 15 minutes now to be safe."
+sleep 900
+
+
+echo 'mopdifiying to provisioned'
+output=$(aws rds modify-db-cluster --db-cluster-identifier $DATABASE_IDENTIFIER --engine-mode provisioned --allow-engine-mode-change --db-cluster-instance-class db.r5.xlarge --apply-immediately --no-paginate )
+PROVISINED_INSTANCE_IDENTIFIER=$(echo $output | jq -r '.DBCluster.DBClusterMembers[0].DBInstanceIdentifier')
```

Comment:
typo but I think it's okay since it looks like it is copied throughout the script